source/features/unreleased-commits.tsx

```diff
@@ -132,7 +132,7 @@ async function addToHome(branchSelector: HTMLButtonElement): Promise<void> {
 		);
 	} else {
 		wrapAll(
-			<div className="d-flex gap-2"/>,
+			<div className="d-flex gap-2" style={{width: "max-content"}}/>,
```

Comment:
It's weird, why is flexbox wrapping? Normally it shouldn't. The right class for this should be `flex-nowrap` or something like that, if they have it, but that's the default for flex box

Reply:
The buttons are in a button group which is a normal div, so that forces it to wrap, we should ideally set width on this group element, but I found it simpler to set this on the branch parent element